webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -123,6 +129,18 @@ export const Table = ({
           )}
           {results.map((result, rowIndex) => (
             <Tr key={rowIndex} ouiaId={`table-row-${rowIndex}`}>
+              {displaySelectAllCheckbox && (
+                <Td
+                  select={{
+                    rowIndex: result.id,
+                    onSelect: (_event, isSelecting) => {
+                      selectOne(isSelecting, result.id);
+                    },
+                    isSelected: isSelected(result.id),
+                    disable: false,
+                  }}
+                />
+              )}
               {columnNamesKeys.map(k => (
```

Comment:
Not sure if this is where the fix is, but the host names need to be links to the host details page :)

Reply:
Fixed!